.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: run pull request CI
+
+on:
+  schedule:
+    - cron:  '0 12 * * *'  # daily ~8am est
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8, 3.9]
```

Comment:
looks like you can add another dimension to this matrix with the tox command to run, e.g.:

```yaml
    command: ["py", "scaffolds"]
```

That would avoid having the (mostly) identical job repetition